asset/script/main.js

```diff
@@ -54,6 +56,56 @@
         }
     };
 
+    function contextOutputControl (element) {
+
+      $(document).on('lint-results', function( event, results) {
+          var inputEl = $(document).find('#post-input')[0];
```

Comment:
I have mixed feelings about the name "inputE1" unless it means something I don't know about.